stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/message/attachments/internal/UploadAttachmentsAndroidWorker.kt

```diff
@@ -51,7 +53,13 @@ internal class UploadAttachmentsAndroidWorker(
         ).uploadAttachmentsForMessage(
             messageId
         ).let { result ->
-            if (result.isSuccess) Result.success() else Result.failure()
+            if (result.isSuccess) {
+                logger.d { "[doWork] Attachments uploaded successfully" }
+                Result.success()
+            } else {
+                logger.i { "[doWork] Error while uploading attachments: ${result.error()}" }
+                Result.failure()
```

Comment:
Shouldn't we return `result` that contains the previous error?

Reply:
it's a `Result` from worker class. We could eventually add `outputData` but not sure if we need it